### DISCUSSION TITLE
Can I throttle async backup speed?

### DISCUSSION THREAD

**Discussion by sgonv** [seaweedfs/seaweedfs#5065] [Q&A] [answered]
Our internet's bandwidth is little bit narrow, so I need to throttle async backup's speed. Is there any way to throttle the speed of async backup? (I run a backup job from seaweedfs server to another seaweedfs server.

**Comment by SmsS4** [comment, marked as answer]
Maybe limit network of backup process with your OS?
For example look at this:
https://serverfault.com/questions/382547/how-to-limit-network-usage-for-concrete-application-in-linux-that-is-running-in

**Reply by sgonv** [reply]
Oh thank you, maybe it will work!
